RPGSystem/RandomNumberGenerator.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace RPGSystem
+{
+    class RandomNumberGenerator
```

Comment:
Put summary descriptions for all classes and their public, protected and internal members (properties, functions, constructors, etc.)
Make the same changes in all classes (excluding generated code)